pkg/appgateserver/metrics.go

```diff
@@ -28,8 +35,18 @@ var (
 	// Usage:
 	// - Monitor health and error rates by service and request type.
 	// - Identify and address high-error areas.
-	relaysErrorsTotal metrics.Counter = prometheus.NewCounterFrom(stdprometheus.CounterOpts{
-		Subsystem: "appgateserver",
-		Name:      "errors_total",
+	relaysErrorsTotal = prometheus.NewCounterFrom(stdprometheus.CounterOpts{
+		Subsystem: appGateServerSubsystem,
+		Name:      relaysErrorsTotalMetric,
+	}, []string{"service_id", "request_type"})
+
+	// relaysSuccessTotal is a Counter metric tracking successful requests on the AppGate server.
+	// Incrementing with each successful request, it's essential for monitoring server
+	// reliability and performance.
+	// Labeled by 'service_id' and 'request_type', it enables detailed analysis
+	// of successful requests.
```

Comment:
We're using requests & relays interchangibly.

Fair to assume the only type of request handled is a relay?

Reply:
Will change to `relay` for consistency and `request_type` to `rpc_type` for clarity